projects/adrv9026/Makefile

```diff
@@ -0,0 +1,7 @@
+###############################################################################
+## Copyright (C) 2023 Analog Devices, Inc.
```

Comment:
The copyright should be 2024 this project

Reply:
Changed copyright to 2023-2024